python/lsst/meas/modelfit/display/interactive.py

```diff
@@ -25,7 +25,6 @@
 import collections
 import numpy
 import copy
-from matplotlib import pyplot
 import matplotlib.colors
 import mpl_toolkits.mplot3d
```

Comment:
The `mpl_toolkits.mplot3d` import actually needs to be moved to `optimizerDisplay.py`, where the `projection='3d'` argument to `add_subplot` requires that module to have been imported (I can see how that would confuse a linter).  Looks like I forgot to move the import when I refactored this code at some point.